ui/src/react-app-env.d.ts

```diff
@@ -1 +1,2 @@
+// eslint-disable-next-line spaced-comment
 /// <reference types="react-scripts" />
```

Comment:
the linter broke up the `///` which caused a surprising failure: svg icons no longer resolved